src/web/handlers/TapGestureHandler.ts

```diff
@@ -26,8 +26,8 @@ export default class TapGestureHandler extends GestureHandler {
   private lastX = 0;
   private lastY = 0;
 
-  private waitTimeout: number | undefined;
-  private delayTimeout: number | undefined;
+  private waitTimeout: NodeJS.Timeout | undefined;
+  private delayTimeout: NodeJS.Timeout | undefined;
```

Comment:
```suggestion
  private waitTimeout: ReturnType<typeof setTimeout> | undefined;
  private delayTimeout: ReturnType<typeof setTimeout> | undefined;
```

Reply:
Done in [3f244d2](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/3f244d27a397c68bf80abedc2c9ed1e450ebf534)